src/User/Controller/SettingsController.php

```diff
@@ -347,9 +354,43 @@ public function actionNetworks()
         );
     }
 
+    public function actionConsent()
+    {
+        $user = Yii::$app->user->identity;
+
+        if (!!$user->gdpr_consent)
```

Comment:
is the double `!` intentional??

Reply:
@Eseperio I think that comment is for you 😄